### ISSUE TITLE
[BUG] Task:  Configure sshd using sshd_config.d] ; Error : AnsibleUndefinedVariable: {{ ansible_user }}: 'ansible_user' is undefined"

### ISSUE THREAD

**Issue by Acenl12** [konstruktoid/ansible-role-hardening#375]
**Describe the bug**
Error : 
**_TASK [konstruktoid.hardening : Configure sshd using sshd_config.d]_** 
`fatal: [localhost]: FAILED! => {"changed": false, "msg": "AnsibleUndefinedVariable: {{ ansible_user }}: 'ansible_user' is undefined"}`

**To Reproduce**
I run this playbook on Debian 11

```
- hosts: localhost
  any_errors_fatal: true
  tasks:
    - name: Include the hardening role
      ansible.builtin.include_role:
        name: konstruktoid.hardening
      vars:
        block_blacklisted: true
        sshd_admin_net:
          - 10.0.2.0/24
          - 192.168.0.0/24
          - 192.168.1.0/24
          - 217.122.0.0/15
        suid_sgid_permissions: false
```

**Expected behavior**

**System (lsb_release -a or similar):**
```
Distributor ID: Debian
Description:    Debian GNU/Linux 11 (bullseye)
Release:        11
Codename:       bullseye
```

**Additional context**

Same issue on Ubuntu 22.04 by the way.

**Comment by konstruktoid** [comment]
Hi @Acenl12, and thanks for opening this issue.

I'll have a PR later today

**Comment by konstruktoid** [comment]
I've merged, https://github.com/konstruktoid/ansible-role-hardening/pull/376. Can you try again?

**Comment by Acenl12** [comment]
Yeah its fixed now, but I'm running in two other errors. Shall I create seperate issues for that?

**Comment by konstruktoid** [comment]
Closed by #376 #377
